src/components/screens/About.js

```diff
@@ -1,37 +1,25 @@
 import PropTypes from 'prop-types';
 import React, { useContext, useState } from 'react';
-import { useQuery } from 'react-apollo';
 import { ActivityIndicator, RefreshControl, SectionList } from 'react-native';
 
 import { colors, texts } from '../../config';
-import { graphqlFetchPolicy } from '../../helpers';
-import { useHomeRefresh, useRefreshTime, useRenderItem } from '../../hooks';
+import { useHomeRefresh, useRenderItem, useStaticContent } from '../../hooks';
 import { NetworkContext } from '../../NetworkProvider';
-import { getQuery, QUERY_TYPES } from '../../queries';
+import { QUERY_TYPES } from '../../queries';
 import { SettingsContext } from '../../SettingsProvider';
 import { LoadingContainer } from '../LoadingContainer';
 import { SectionHeader } from '../SectionHeader';
 import { VersionNumber } from '../VersionNumber';
 
 export const About = ({ navigation, withHomeRefresh, withSettings }) => {
-  const { isConnected, isMainserverUp } = useContext(NetworkContext);
+  const { data: aboutData, loading, refetch } = useStaticContent({
+    name: 'homeAbout',
+    type: 'json'
```

Comment:
should there be a `refreshTimeKey` with `'publicJsonFile-homeAbout'` here, because otherwise the `name` will be used and could lead to different behaviors than before?

Reply:
this would lead to users possibly needing to fetch one extra time after updating the app. i changed it to use the `refreshTimeKey` as suggested.

Reply:
which would be fine one extra time.
but not necessary to do "something else than before".
you know how we like it :)